src/model/__init__.py

```diff
@@ -0,0 +1,5 @@
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+from .user import User
```

Comment:
As I understand, all new models need to be added here. In such a case, please add the comment with explanation here.